lib/Varien/Db/Tree/NodeSet.php

```diff
@@ -58,13 +58,10 @@ public function valid(): bool
         return isset($this->_nodes[$this->_current]);
     }
 
-    #[\ReturnTypeWillChange]
     public function next()
```

Comment:
Also need to be compatible with method `Iterator::next()`:
```suggestion
    public function next(): void
```

![image](https://github.com/OpenMage/magento-lts/assets/1106470/67d8bac6-d4b4-4a7e-a49f-e37b001bfc56)